guillotina/contrib/swagger/services.py

```diff
@@ -91,11 +91,11 @@ def get_endpoints(self, iface_conf, base_path, api_def, tags=None):
         for method in iface_conf.keys():
             if method == "endpoints":
                 for name in iface_conf["endpoints"]:
+                    for http_method in iface_conf["endpoints"][name].keys():
+                        if "tags" in iface_conf["endpoints"][name][http_method]:
+                            tags = iface_conf["endpoints"][name][http_method]["tags"]
```

Comment:
Does it make sense to add a `break` after we find the tags? Or we want to assign the latest value?

Reply:
yeah i think we only want the first match